src/components/screens/all-software-screen.tsx

```diff
@@ -35,36 +36,38 @@ import SearchIcon from "@mui/icons-material/Search";
 import useCreateSoftware from "src/hooks/use-create-software";
 import AddSoftwareModal from "../software-registry/AddSoftwareModal";
 
-const statusOptions = [
-  { value: "ALL", label: "Show all" },
-  { value: "PENDING", label: "Pending" },
-  { value: "UNDER_REVIEW", label: "Under review" },
-  { value: "ACCEPTED", label: "Accepted" },
-  { value: "DEPRECATED", label: "Deprecated" },
-  { value: "DECLINED", label: "Declined" },
-];
+
 
 /**
  * All software screen component
  */
-const AllSoftwareScreen: React.FC = () => {
+const AllSoftwareScreen = () => {
   const { softwareApi } = useLambdasApi();
-  const [applications, setApplications] = useState<SoftwareRegistry[]>([]);
-  const [loading, setLoading] = useState<boolean>(true);
+  const [software, setApplications] = useState<SoftwareRegistry[]>([]);
+  const [loading, setLoading] = useState(true);
```

Comment:
Typically we want to initialize loading states as false and only set them to true when an async request is being made. I'd suggest changing this appropriately.

Reply:
You will need to check all the requests if there was one that was expecting loading to be true whereas we need to make sure it's being set at the appropriate time e.g. line 84 is correct in fetchSoftwareData